etc/ReplayOfflineConfiguration.py

```diff
@@ -100,7 +100,7 @@
 
 # Defaults for CMSSW version
 defaultCMSSWVersion = {
-    'default': "CMSSW_11_3_2"
+    'default': "CMSSW_12_1_0_pre3"
```

Comment:
@qliphy shouldn't you also add the cmssw override for repacking (line 145) and express (line 159)?

Reply:
@francescobrivio   Thanks. Now added.